src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Random random = new Random();
+    private ColorSupplier csp = new ColorSupplier();
+    private final int maxNumber = 100;
```

Comment:
naming convention:
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names

Reply:
Not fixed. Constants should be in UPPERCASE